Tasks/Common/utility-common/package-lock.json

```diff
@@ -9,7 +9,7 @@
       "resolved": "https://registry.npmjs.org/argparse/-/argparse-1.0.10.tgz",
```

Comment:
changes in this file doesn't seem related, should we ignore it?

Reply:
Yes. These won't be checked in.